src/server/src/organizations/services/organizations.service.ts

```diff
@@ -20,39 +20,17 @@ export class OrganizationsService extends TypeOrmCrudService<Organization> {
     // Returns public data for organization screen
     const builder = this.repo.createQueryBuilder("organization");
     const data = await builder
-      .leftJoinAndSelect("organization.users", "users")
-      .leftJoinAndSelect("organization.admin", "admin")
+      .select()
+      .leftJoin("organization.users", "users")
+      .leftJoin("organization.admin", "admin")
```

Comment:
Wondering if this would be more readable if I move the SELECTs up to the tables that match them - @kevinearldenny what do you think?
```suggestion
      .leftJoin("organization.users", "users")
      .addSelect(["users.id", "users.name"])
      .leftJoin("organization.admin", "admin")
      .addSelect(["admin.id", "admin.name"])
```